zerver/webhooks/sentry/doc.md

```diff
@@ -12,28 +12,27 @@ us](/help/contact-support) if a platform you care about is missing.
     issue or event.
 
 3. Go to your organization's **settings** in Sentry. Then go to
-**Developer Settings** and click on the button to create a
-**New Internal Integration**. There, set the **Webhook URL** to
-the URL you constructed in the above step then enable
+**Developer Settings** and click on the **Create New Integration**
+button, then select **Internal Integration**. There, set the
+**Webhook URL** to the URL you constructed in the above step then enable
```

Comment:
This commit would ideally have a commit message explaining the nature of the changes a bit more -- some at least sound like "Sentry's UI has changed", and that detail would be nice to call out.